tools/mapmerge/map_helpers.py

```diff
@@ -123,7 +123,7 @@ def merge_map(newfile, backupfile, tgm):
         write_grid_coord_small(newfile, mergeGrid)
     else:
         write_dictionary(newfile, originalDict)
-        write_grid(newfile, originalDict)
+        write_grid(newfile, mergeGrid)
```

Comment:
Yea that's how it was supposed to be. It's currently bugged if you want to write a normal dmm map.